media/editor/findWidget.tsx

```diff
@@ -234,6 +234,7 @@ export const FindWidget: React.FC = () => {
 				cap: isUncapped ? undefined : defaultResultCap,
 				query: queryOrError,
 				caseSensitive: isCaseSensitive,
+				binaryMode: isBinaryMode,
```

Comment:
Thanks for the PR! This could just be

```suggestion
				caseSensitive: isBinaryMode || isCaseSensitive,
```

...and the other files would not need to change

Reply:
Oh, yeah. You're right. I didn't reailize I could just do that.